Source/WebCore/dom/Node.cpp

```diff
@@ -2220,26 +2220,36 @@ void Node::moveNodeToNewDocument(Document& oldDocument, Document& newDocument)
             newDocument.didAddEventListenersOfType(eventType, count);
         });
 
+        unsigned numWheelEventHandlers = 0;
```

Comment:
Would be nice to avoid the abbreviation in the variable names here for `num`, either by using `numberOf` as prefix or by using `Count` as suffix.

Reply:
I'm just moving this declaration. If we rename it, we'd have to touch more lines.

Reply:
Ah ok. No biggie.